### PR TITLE
Hackathon event July2020

### PR DESCRIPTION
**Pull request by ggabernet**
- Add July2020 Hackathon event

### CODE REVIEW

**Comment by ggabernet** [comment]
Still needs the link to the DKFZ registration

File: markdown/events/2020/hackathon-july-2020.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+where they will be also be available afterwards. Discussion and collaboration
+will be handled in dedicated channels in the nf-core Slack organisation.
+
+Relevant links will be emailed to participants ahead of the event. For more
+details or if you have questions, please post a message in the nf-core Slack.
+See [https://nf-co.re/join](https://nf-co.re/join) for information on how to join.
+All nf-core code is open source with the MIT licence and available on
+GitHub: [https://github.com/nf-core](https://github.com/nf-core)
+
+This event is being organised by the nf-core core team.
```
Comment (ewels) [inline markdown/events/2020/hackathon-july-2020.md:49]:
```suggestion
This event is being organised by the nf-core [core team](https://nf-co.re/about).
```

File: markdown/events/2020/hackathon-july-2020.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+To participate, all you need is a computer with an internet connection.
+Presentations will be run using Zoom and live-streamed to YouTube,
+where they will be also be available afterwards. Discussion and collaboration
+will be handled in dedicated channels in the nf-core Slack organisation.
+
+Relevant links will be emailed to participants ahead of the event. For more
+details or if you have questions, please post a message in the nf-core Slack.
+See [https://nf-co.re/join](https://nf-co.re/join) for information on how to join.
+All nf-core code is open source with the MIT licence and available on
```
Comment (ewels) [inline markdown/events/2020/hackathon-july-2020.md:46]:
```suggestion

All nf-core code is open source with the MIT licence and available on
```

File: markdown/events/2020/hackathon-july-2020.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+GitHub: [https://github.com/nf-core](https://github.com/nf-core)
+
+This event is being organised by the nf-core core team.
+
+## Hackathon project page
+
+[https://github.com/orgs/nf-core/projects/6](https://github.com/orgs/nf-core/projects/6)
+
+## Registration
```
Comment (ewels) [inline markdown/events/2020/hackathon-july-2020.md:55]: I think this should be moved further up, maybe after the introduction, before the schedule?

File: markdown/events/2020/hackathon-july-2020.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+can contribute to nf-core and get your pipeline started
+from the comfort of your homes.
+
+Join us to get help and guidance for writing your own Nextflow pipelines
+using the best practice guidelines implemented by the nf-core community
+or contribute to existing pipelines and advancing the current infrastructure.
+This will be a very informal event and it's great for learning,
+sharing ideas and solving issues intensively.
+
+## Schedule
```
Comment (ewels) [inline markdown/events/2020/hackathon-july-2020.md:21]: If the schedule is now fixed, we could add a Markdown table here with the full schedule.

https://www.tablesgenerator.com/markdown_tables

File: markdown/events/2020/hackathon-july-2020.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
+---
+title: Hackathon July 2020
+subtitle: our first fully remote hackathon
```
Comment (ewels) [inline markdown/events/2020/hackathon-july-2020.md:3]:
```suggestion
subtitle: The first online-only nf-core hackathon
```

File: markdown/events/2020/hackathon-july-2020.md (hunk 3 above)
Comment (ggabernet) [reply markdown/events/2020/hackathon-july-2020.md:56]: ok, yes makes sense!

**Pull request merged**